### PR TITLE
D54 - Change cluster icon style, and change full zoom button position

### PR DESCRIPTION
**Pull request by tienhoah**

### CODE REVIEW

**Comment by tienhoah** [comment]
closes #54

File: src/components/Button.js [hunk 1]
```diff
@@ -17,13 +16,18 @@ export const ButtonStyle = styled('button')`
   position: ${props => props.theme.position};
   padding: ${props => props.theme.padding};
   width: ${props => props.theme.width};
-  box-shadow: ${props => props.theme.shadow};
+  height: ${props => props.theme.height};
```
Comment (brianbancroft) [inline src/components/Button.js]: Will this height be used for buttons, or for other things as well? Is the root of the theme a good place for it?

**Comment by brianbancroft** [comment]
I get the feeling that the cluster icon is off-center. Also, what happens when there's more than ten items in the cluster? 
image

File: src/components/LeafletMap.js [hunk 2]
```diff
@@ -55,6 +55,20 @@ const activeInnerMarkerHtmlStyles = `
   border-top: 15px solid ${defaultMarkerColor};
 `;
 
+const clusterIconStyles = `
```
Comment (brianbancroft) [inline src/components/LeafletMap.js]: When it comes to centerring things, I tend to cheap out and use flex:

```
display: flex;
justify-content: center;
align-items: center;

```

File: src/components/LeafletMap.js [hunk 3]
```diff
@@ -55,6 +55,20 @@ const activeInnerMarkerHtmlStyles = `
...
+  width: 24px;
+  height: 24px;
```
Comment (brianbancroft) [inline src/components/LeafletMap.js]: Making a tagged litteral higher-order function might be the way to go here, if you want to make the size and shape (or even the colour) of the cluster variable in itself. I'm going to try to set up an example.

Comment (brianbancroft) [reply src/components/LeafletMap.js]: I'm going to talk about this in #javascript, because this is a tricky topic that involves higher order functions.

**Pull request merged**